main/ejml-dsparse/src/org/ejml/sparse/csc/CommonOpsWithSemiRing_DSCC.java

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright (c) 2009-2020, Peter Abeles. All Rights Reserved.
+ *
+ * This file is part of Efficient Java Matrix Library (EJML).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ejml.sparse.csc;
+
+import org.ejml.MatrixDimensionException;
+import org.ejml.data.DGrowArray;
+import org.ejml.data.DMatrixRMaj;
+import org.ejml.data.DMatrixSparseCSC;
+import org.ejml.data.IGrowArray;
+import org.ejml.ops.DSemiRing;
+import org.ejml.sparse.csc.misc.ImplCommonOpsWithSemiRing_DSCC;
+import org.ejml.sparse.csc.mult.ImplSparseSparseMultWithSemiRing_DSCC;
+
+import javax.annotation.Nullable;
+
+import static org.ejml.UtilEjml.reshapeOrDeclare;
+import static org.ejml.UtilEjml.stringShapes;
+
+
+public class CommonOpsWithSemiRing_DSCC {
+
+    public static DMatrixSparseCSC mult(DMatrixSparseCSC A, DMatrixSparseCSC B, @Nullable DMatrixSparseCSC output, DSemiRing semiRing) {
+        return mult(A, B, output, semiRing, null, null);
+    }
+
+    /**
+     * Performs matrix multiplication.  output = A*B
+     *
+     * @param A        (Input) Matrix. Not modified.
+     * @param B        (Input) Matrix. Not modified.
+     * @param output   (Output) Storage for results.  Data length is increased if increased if insufficient.
```

Comment:
Typo: `is increased if increased`

Reply:
this java-doc is just copied. I will fix it in both places